acados/ocp_nlp/ocp_nlp_common.c

```diff
@@ -1106,6 +1107,9 @@ void ocp_nlp_opts_initialize_default(void *config_, void *dims_, void *opts_)
     // globalization
     opts->alpha_min = 0.05;
     opts->alpha_reduction = 0.7;
+    opts->full_step_dual = 0;
+    opts->line_search_use_sufficient_descent = 0;
+    opts->glob_SOC = 0;
```

Comment:
Same for this: SOC is second order correction, but what is `glob`?

Reply:
I renamed the option to `globalization_use_SOC`.
I guess that is more clear.